src/disco/topo/fd_topo.h

```diff
@@ -205,6 +205,8 @@ typedef struct {
       char  snapshot[ PATH_MAX ];
       char  incremental[ PATH_MAX ];
       char  capture[ PATH_MAX ];
+      char  vote_account_path[ PATH_MAX ];
```

Comment:
Is this the authorized voter? What is this?

Reply:
No, authorized voter keypair is the same as validator identity in our case. If you do `solana validators -ul`, you can see 2 pubkeys printed for each validator: `Identity` and `Vote Account`. This `vote_account_path` here is the `Vote Account` printed.

Reply:
Can you add a TODO to make this param a base58 string instead of a file?